ifupdown2/addons/vxlan.py

```diff
@@ -393,7 +393,7 @@ def __config_vxlan_local_tunnelip(self, ifname, ifaceobj, link_exists, user_requ
         if link_exists:
             # on ifreload do not overwrite anycast_ip to individual ip
             # if clagd has modified
-            running_localtunnelip = cached_vxlan_ifla_info_data.get(Link.IFLA_VXLAN_LOCAL)
```

Comment:
The function `__config_vxlan_local_tunnelip` should be reworked so that it handles both `vxlan-local-tunnelip` and `vxlan-local-tunnelip6`. 

It could take the NETLINK_ATTR id as parameter (`Link.IFLA_VXLAN_LOCAL` or `Link.IFLA_VXLAN_LOCAL6`), as well as the attribute name so that no other changes are needed (this is what i should have done for mcastgrp6).